vitrina/orgs/models.py

```diff
@@ -143,8 +143,8 @@ class Meta:
 
 
 class Representative(models.Model):
-    COORDINATOR = "coordinator"
-    MANAGER = "manager"
+    COORDINATOR = "coordinator"  # Tvarkytojas + tvarko kitus Representative
+    MANAGER = "manager"  # Tvarkytojas
```

Comment:
Since we code in english and all other comments are in english, let's correct these comments to english as well